### PR TITLE
add support for paths

### PR DESCRIPTION
**Pull request by miguelgarcia**
* Add support for paths that are calculated from the trajectories in edgeStates

### CODE REVIEW

**Comment by miguelgarcia** [comment]
@leandropineda can you review ?

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ is sent to InOrbit.
 
 The service reads its configuration from the following environment variables:
 
-* `INORBIT_APP_KEY`: InOrbit App Key.
+* `INORBIT_API_KEY`: InOrbit App Key.
```
Comment (leandropineda) [inline README.md:15]:
```suggestion
* `INORBIT_API_KEY`: InOrbit API Key.
```

File: demo/index.js [hunk 2]
```diff
@@ -136,7 +136,22 @@ async function main () {
...
-            edgeStates: [],
+            edgeStates: [
+                // Example trajectory that translates to a path
+                {
+                    edgeId: '0',
+                    sequenceId: 0,
+                    released: true,
+                    trajectory: {
+                        degree: 1,
+                        knotVector: [0],
```
Comment (leandropineda) [inline demo/index.js:147]: I don't think it really matters at this point but the documentation says that `knotVector has size of number of control points + degree + 1`. I used `[0, 0, 0, 0]` on my synthetic sample for 2 control points

File: src/vda5050ToIOProxy.js [hunk 3]
```diff
@@ -169,4 +170,39 @@ export default class VDA5050ToInOrbitProxy {
...
+          // edge in base, not-yet-traversed. Let's use the first and last control points
+          // in the trajectory to build the path
+          // TODO(mike) we are ignoring all the complexities about working with NURBS
+          // check the standard for more details about how trajectories are represented
+          const { controlPoints } = e.trajectory;
+          const start = controlPoints[0];
+          const end = controlPoints[controlPoints.length - 1];
+          if (points.length == 0 || (
+            points[points.length - 1].x != start.x
+            && points[points.length - 1].y != start.y
```
Comment (leandropineda) [inline src/vda5050ToIOProxy.js:196]: I think that for checking if the last point is different you should use x `OR` y  (or both) changed.

File: demo/index.js (hunk 2 above)
Comment (miguelgarcia) [reply demo/index.js:147]: changed!

**Comment by miguelgarcia** [comment]
I addressed your feedback @leandropineda , can you re-review?

**Review by leandropineda — APPROVED** [review]

**Pull request merged**